### PR TITLE
added dispute endpoint

### PR DESCRIPTION
**Pull request by ankitdas13**
Api doc reference

[Fetch All Disputes](https://razorpay.com/docs/api/disputes/fetch-all)
[Fetch a Dispute](https://razorpay.com/docs/api/disputes/fetch)
[Accept a Dispute](https://razorpay.com/docs/api/disputes/accept)
[Contest a Dispute](https://razorpay.com/docs/api/disputes/contest)

### CODE REVIEW

File: src/test/java/com/razorpay/DisputeClientTest.java [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+    /**
+     * Contest a Dispute using dispute id.
+     * @throws RazorpayException
+     */
+    @Test
+    public void TestContest() throws RazorpayException{
+
+        JSONObject request = new JSONObject("{\n" +
+                "  \"amount\": 5000,\n" +
```
Comment (rohitcbr) [inline src/test/java/com/razorpay/DisputeClientTest.java:26]: As passing a string to JSONObject looks untidy, please do it the same way it is done here https://github.com/razorpay/razorpay-java/blob/55a19372e8aeb9c74cbdabd8286356252a0304a2/src/test/java/com/razorpay/CustomerClientTest.java#L308

**Comment by rohitcbr** [comment]
Some checks are failing and skipped, please check it.

**Comment by ankitdas13** [comment]
@rohitcbr for last check, I have to mark this pr as `Tested / TestingNotRequired` Once this pr is approved, I will mark this.

**Comment by ankitdas13** [comment]
@rohitcbr for last check, I have to mark this pr as `Tested / TestingNotRequired` Once this pr is approved, I will mark this.

**Review by rohitcbr — APPROVED** [review]

**Pull request merged**